docs/src/userguide/iris_philosophy.rst

```diff
@@ -0,0 +1,191 @@
+.. _iris-philosophy:
+
+****************
+Iris' Philosophy
+****************
+
+.. todo:: https://github.com/SciTools/iris/issues/6511; this page belongs in 'Explanation'
+
+.. _code-maintenance:
+
+Code Maintenance
+================
+
+From a user point of view "code maintenance" means ensuring that your existing
+working code stays working, in the face of changes to Iris.
+
+
+Stability and Change
+---------------------
+
+In practice, as Iris develops, most users will want to periodically upgrade
+their installed version to access new features or at least bug fixes.
+
+This is obvious if you are still developing other code that uses Iris, or using
+code from other sources.
+However, even if you have only legacy code that remains untouched, some code
+maintenance effort is probably still necessary:
+
+* On the one hand, *in principle*, working code will go on working, as long
+  as you don't change anything else.
+
+* However, such "version stasis" can easily become a growing burden, if you
+  are simply waiting until an update becomes unavoidable, often that will
+  eventually occur when you need to update some other software component,
```

Comment:
Grammar suggestion: unnecessary comma here?
```suggestion
  eventually occur when you need to update some other software component
```

or start the phrase with something like "perhaps", or "potentially" e.g.:

> eventually occur when you need to update some other software component, potentially for some some completely unconnected reason

Reply:
824bda2c9